src/Access/MediaAvailableAccess.php

```diff
@@ -133,12 +135,20 @@ protected function mergeCacheMaxAge(MpxMedia $mpx_media, MediaInterface $media)
     $available_date = $mpx_media->getAvailableDate();
     if ($available_date instanceof ConcreteDateTime &&
       $now < $available_date->getDateTime()) {
-      $media->mergeCacheMaxAge($available_date->getDateTime()->getTimestamp() - $now->getTimestamp());
+      $delta = $available_date->getDateTime()->getTimestamp() - $now->getTimestamp();
+      // Safe guard against radically far out dates. Set the max age to the min
```

Comment:
This is amazing. In particular that you've managed to use the word "radical" in such a "far out" PR.

Reply:
Ha! 👍